app/actions/nodeStorageActions.js

```diff
@@ -110,7 +110,9 @@ export const getRPCEndpoint = async (
 }
 
 export const getNode = async (net: Net): Promise<string> => {
-  const storage = await getStorage(`${STORAGE_KEY}-${net}`).catch(console.error)
+  const storage = await getStorage(`${STORAGE_KEY}-${net}`).catch(e =>
+    console.error(e),
+  )
```

Comment:
Fine with this change but if Im not mistaken its the same as what was there previously

Reply:
Oh sorry, I wasn't 100% sure 😄